digiwf-integrations/digiwf-example-integration/digiwf-example-integration-starter/src/main/java/de/muenchen/oss/digiwf/example/integration/ExampleIntegrationAdapterInAutoconfiguration.java

```diff
@@ -0,0 +1,32 @@
+package de.muenchen.oss.digiwf.example.integration;
+
+import de.muenchen.oss.digiwf.example.integration.core.adapter.in.streaming.ExampleDto;
+import de.muenchen.oss.digiwf.example.integration.core.adapter.in.streaming.ExampleMapper;
+import de.muenchen.oss.digiwf.example.integration.core.adapter.in.streaming.MessageProcessor;
+import de.muenchen.oss.digiwf.example.integration.core.application.port.in.ExampleInPort;
+import de.muenchen.oss.digiwf.example.integration.core.application.port.in.ProcessResponseInPort;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.messaging.Message;
+
+import java.util.function.Consumer;
+
+@Configuration
+public class ExampleIntegrationAdapterInAutoconfiguration {
+
+    @ConditionalOnMissingBean
+    @Bean
+    public MessageProcessor messageProcessor(
+            ProcessResponseInPort processResponseInPort,
+            ExampleInPort exampleInPort,
+            ExampleMapper exampleMapper
+    ) {
```

Comment:
Nur damit es einheitlich zur Bean unten ist.

```suggestion
    public MessageProcessor messageProcessor(
            final ProcessResponseInPort processResponseInPort,
            final ExampleInPort exampleInPort,
            final ExampleMapper exampleMapper
    ) {
```

Reply:
Ist korrigiert und hab eine andere AutoConfig auch noch angepasst.